### DISCUSSION TITLE
Chatwoot enchendo espaço da VPS

### DISCUSSION THREAD

**Discussion by felipemendesjfora** [chatwoot/chatwoot#10492] [Q&A] [answered]
Estou comum problema em que constantemente meu chatwoot fica offline por problemas de espaço em disco na VPS. Faço Limpezas no Linux mas não está resolvendo. Como posso proceder?

**Comment by jtayrone** [comment, marked as answer]
Você precisa ajustar seu disco para uma capacidade maior. Além disso sugiro usar um storage externo.

Cloud Storage Support
https://www.chatwoot.com/docs/self-hosted/deployment/storage/supported-providers
